crates/context/src/lib.rs

```diff
@@ -283,172 +271,171 @@ impl ContextManager {
             eyre::bail!("Context not found")
         };
 
-        value.application_id = application_id.0.into();
+        value.application = calimero_store::key::ApplicationMeta::new(application_id);
 
         handle.put(&key, &value)?;
 
         Ok(())
     }
+}
 
-    pub async fn list_installed_applications(
+// vv~ these would be more appropriate in an ApplicationManager
```

Comment:
highlighting for visibility

Reply:
What is "vv"? 

 - this and other similar comments should be converted to the issues as otherwise they will get lost

Reply:
`^`, `<`, `>`, `v`

arrow pointing down